### PR TITLE
Added Pulkit to Contestants

### PR DESCRIPTION
**Pull request by Tachyon01**
Added Pulkit

### CODE REVIEW

File: contestants.json [hunk 1]
```diff
@@ -223,6 +223,15 @@
...
 	   
+      },
+	{
+	 "name":"Pulkit Goel",
+         "imageurl":"contestant/pulkit.jpeg",
+         "about":"JYup, that's my profile pic",
+         "facebook":"",
+         "github":"https://github.com/Tachyon01",
+         "twitter":""
+
```
Comment (pradeepgangwar) [inline contestants.json]: Remove extra line here

**Comment by pradeepgangwar** [comment]
Also make meaningful PR titles and commit messages. Like `Add Pulkit to list of cotestents`.

File: contestants.json (hunk 1 above)
Comment (Tachyon01) [reply contestants.json]: Done

**Comment by pradeepgangwar** [comment]
Don't delete facebook and twitter. Just leave them blank like before. I asked for unnecessary empty line.

File: contestants.json [hunk 2]
```diff
@@ -223,6 +223,14 @@
          "github":"https://github.com/amriteshbhaskar",
          "twitter":""
 	   
+      },
+	{
+	 "name":"Pulkit Goel",
+         "imageurl":"contestant/pulkit.jpeg",
+         "about":"Yup, that's my profile pic",
+         "facebook":"",
+	 "github":"https://github.com/Tachyon01"
```
Comment (pradeepgangwar) [inline contestants.json]: Put a `,` at the end of this line.

**Comment by pradeepgangwar** [comment]
@Tachyon01 I fixed your merge conflicts this time. But please correct them yourself before making a PR.

**Comment by pradeepgangwar** [comment]
@Tachyon01 You deleted others record. Please restore their record and update your record.

File: contestants.json [hunk 3]
```diff
@@ -221,17 +221,15 @@
...
+         "twitter":""	   
       },
       {
-	 "name":"Tanmay Mittal",
-         "imageurl":"contestant/tanmay.jpeg",
-         "about":"New to this Developer's World",
-         "facebook":"https://www.facebook.com/tanmay.mittal.106",
-         "github":"https://github.com/strikertanmay",
-         "twitter":""
-
```
Comment (pradeepgangwar) [inline contestants.json]: Why deleting this record?

**Comment by pradeepgangwar** [comment]
@Tachyon01 Next time beware of merge conflicts. This time i fixed for you.

**Pull request merged**